packages/components/src/internal/components/domainproperties/dataclasses/constants.ts

```diff
@@ -0,0 +1,16 @@
+export const DATACLASS_DOMAIN_SYSTEM_FIELDS = [
+    {
+        Name: 'Name',
+        Label: 'Name',
+        'Data Type': 'Text',
+        Required: false,
+        Description: 'Contains a short name for this data object.',
+    },
+    {
+        Name: 'Description',
+        Label: 'Description',
+        'Data Type': 'Text',
+        Required: false,
+        Description: 'Contains a description for this data object',
```

Comment:
this is probably just taken from the schema browser, but either this desc should end with a period or the other one should not.

Reply:
Good point—took out all trailing periods in the constants